### PR TITLE
Pydra

### PR DESCRIPTION
**Pull request by wilke0818**
- Imports pydra serialization for tensors
- Currently has the serializer in 2 places, the \_\_init__ and in a helper function module, should choose one and use that
- Has some miscelanrous bug fixes for audio classification

### CODE REVIEW

File: src/senselab/__init__.py [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+import nest_asyncio
+import numpy as np
+import torch
+from pydra.utils.hash import Cache, bytes_repr_sequence_contents, register_serializer
+
+nest_asyncio.apply()
+
+
+@register_serializer(torch.Tensor)
```
Comment (fabiocat93) [inline src/senselab/__init__.py]: hey @wilke0818 why is the `bytes_repr_arraylike` both here and in the `pydra_helpers.py`?

Comment (wilke0818) [reply src/senselab/__init__.py]: well that's part of the question I had: where do we want this to live? If we leave it in `pydra_helpers.py` then we need to import it in the `init`. I wasn't sure which made more sense especially as I am not sure if we will have more pydra related utilities? You can kind of see the notes I was making for what a Pydra class could look like.

Comment (fabiocat93) [reply src/senselab/__init__.py]: i like the idea of having it in pydra_helpers and import it in the init. we will probably need something similar for setting up global settings for huggingface and other libraries at some point. just make a decision and let's see how it works for now. we can always change if we realize that there are better options

Comment (wilke0818) [reply src/senselab/__init__.py]: done. we should look at using \_\_all__ though I don't fully understand it/would require a bit more restructuring than I was originally intending with this PR

**Review by fabiocat93 — APPROVED** [review]

**Pull request merged**